apps/projects/app/components/Content/Settings.js

```diff
@@ -33,6 +32,8 @@ class Settings extends React.Component {
     onLogin: PropTypes.func.isRequired,
     status: PropTypes.string.isRequired,
     tokens: PropTypes.array.isRequired,
+    layoutName: PropTypes.string.isRequired,
+    theme: PropTypes.object,
```

Comment:
With the current code, it makes sense to make `theme` a required prop, otherwise it will fail silently or lead to unexpected issues

Reply:
This has been converted to functional :)